tdrs-backend/tdpservice/urls.py

```diff
@@ -5,18 +5,11 @@
 from django.contrib import admin
 from django.urls import include, path, re_path, reverse_lazy
 from django.views.generic.base import RedirectView
-
-from rest_framework.routers import DefaultRouter
-
 from .users.api.authorization_check import AuthorizationCheck
 from .users.api.login import TokenAuthorizationOIDC
 from .users.api.login_redirect_oidc import LoginRedirectOIDC
 from .users.api.logout import LogoutUser
 from .users.api.logout_redirect_oidc import LogoutRedirectOIDC
-from .users import views
-
-router = DefaultRouter()
-router.register("users", views.UserViewSet)
```

Comment:
We moved the users urls to the app to be consistent with how the rest of the application will handle urls